.github/workflows/maestro_mcp_tool_test.yaml

```diff
@@ -0,0 +1,111 @@
+name: Maestro Create tool Test
+on:
+  push:
+    branches: [ "main" ]
+    paths:
+    - '**'
+  pull_request:
+    branches: [ "main" ]
+    paths:
+    - '**'
+jobs:
+  test-in-kind:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Install Kind
+      run: |
+        curl -Lo ./kind "https://github.com/kubernetes-sigs/kind/releases/download/v0.20.0/kind-$(uname)-amd64"
+        chmod +x ./kind
+        sudo mv kind /usr/local/bin
+        which kind
+    - name: Create Kind cluster
+      run: |
+        kind create cluster --config tests/integration/deploys/kind-config.yaml 
```

Comment:
Did you check if this could be done with a published action? https://github.com/marketplace/actions/kind-cluster

Reply:
Yes, I did.  it doesn't have the necessary config option.